Azrux/Athenas-front
# Lau/create landing page

## Description
Landing page was added, it includes:
* Main section with background photo and a call to action button to visit Home page
* About Athenas section with 3 collapsible sections for:
  * Why choose us?
  * Testimonials
  * Our products
* Form section to sign up (not working)
* Contact us section with a map

## How to test?
* Run npm i
* Run npm run dev
* Check design and responsive design
* Check that every functionality is working fine
